src/components/KanaTable/BasicKanaTable.tsx

```diff
@@ -0,0 +1,76 @@
+import KanaType from '../../types/kana';
+import { Caption, Table } from './KanaTableStyles';
+import TableHead from './TableHead';
+import TableRow from './TableRow';
+
+type BasicKanaTableProps = {
+  kana: KanaType[];
+  caption: string;
+};
+
+export default function BasicKanaTable({ kana, caption }: BasicKanaTableProps) {
+  const yKana = kana.filter(({ group }) => group === 'y');
+  const newYKanaArray: KanaType[] = [
+    yKana[0],
+    { id: '1', pronunciation: '', kana: '', type: 'basic', group: '' },
+    yKana[1],
+    { id: '2', pronunciation: '', kana: '', type: 'basic', group: '' },
+    yKana[2],
+  ];
```

Comment:
Because I need to change the length of the array to match 5 elements. With `.map()` I only am able to return an array of the same length - if I am not mistaken 🙈

Reply:
Oh yes you are right, and I also saw that you need the objects in certain places for the table.